cmd/agent/install_script.sh

```diff
@@ -6,6 +6,7 @@
 # using the package manager and Datadog repositories.
 
 set -e
+install_script_version=7.19.0
```

Comment:
Does that mean we'll start releasing the install script with the Agent? 
Otherwise we shouldn't use the same exact versioning as the Agent, as the install script "releases" are currently independent from Agent releases.

Reply:
I have no strong opinion on that. I guess it could have its own version.

Reply:
+1 on having its own version. We could go with 1.0, although if we do semver we won't ever get to 2.0 (because we can't break backwards compat).